packages/preview/ofbnote/0.2.0/template/main.typ

```diff
@@ -0,0 +1,22 @@
+#import "@local/ofbnote:0.2.0": *
```

Comment:
Can you please update that to use the final package name?

```suggestion
#import "@preview/ofbnote:0.2.0": *
```

Reply:
Yes sorry, it is done.